### ISSUE TITLE
https://sfg-taxonpages.github.io/orthoptera/#/otus/807411/overview

### ISSUE THREAD

**Issue by klausriede** [sfg-taxonpages/orthoptera#28]
A. cheesmanae does not appear as in http://orthoptera.speciesfile.org/Common/basic/Taxa.aspx?TaxonNameID=1101290 (originally apterotettix chesmanae ?)

**Comment by klausriede** [comment]
the junior synonym history of kokodensis only becomes clear in old OSF http://orthoptera.speciesfile.org/Common/basic/Taxa.aspx?TaxonNameID=1101291

**Comment by typophyllum** [comment]
I agree ... perhaps the names of the synonyms could be made active to lead to a display similar to Browse OTU when set to "Current name". Then at least their taxonomic history would be clearly visible.

![imagen](https://github.com/sfg-taxonpages/orthoptera/assets/33210201/e19e3de2-95f3-48d5-9d58-21d8e4275631)
